src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -849,18 +849,23 @@ public final void markDirty()
      * register a block and position.
      *
      * @param block to be registered
-     * @param pos of the block
+     * @param pos   of the block
+     * @param world the world to place the tileentity in
      */
-    public void registerBlockPosition(@NotNull Block block, @NotNull final BlockPos pos)
+    public void registerBlockPosition(@NotNull Block block, @NotNull final BlockPos pos, @NotNull final World world)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "block". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)